cmake/f3dPlugin.cmake

```diff
@@ -263,4 +283,23 @@ macro(f3d_plugin_build)
       EXCLUDE_FROM_ALL)
   endforeach()
 
+  # JSON
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "name" "\"${F3D_PLUGIN_NAME}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "description" "\"${F3D_PLUGIN_DESCRIPTION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "version" "\"${F3D_PLUGIN_VERSION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "type" "\"${F3D_PLUGIN_TYPE}\"")
+
+  file(WRITE "${CMAKE_BINARY_DIR}/${F3D_PLUGIN_NAME}.json" "${F3D_PLUGIN_JSON}")
+
+  install(FILES "${CMAKE_BINARY_DIR}/${F3D_PLUGIN_NAME}.json"
+      DESTINATION "share/f3d/plugins"
+      COMPONENT json
+      EXCLUDE_FROM_ALL)
```

Comment:
So this needs to be installed manually ? I think this should be part of the sdk component.

Reply:
ok makes sense